.firebaserc

```diff
@@ -1,5 +1,25 @@
 {
   "projects": {
-    "default": "nwplus-ubc-dev"
+    "default": "nwplus-ubc-dev",
+    "production": "nwplus-ubc"
+  },
+  "targets": {
+    "production": {
```

Comment:
Why is line 7 "production" but line 17 is "nwplus-ubc-dev"

Reply:
Firebase automatically configured the `.firebaserc` file when I set up the targets for multi-site hosting. I don't know why it used the alias for prod but the actual name for the dev database. I will see if manually changing it still works

Reply:
A quick update on this: I tried changing the rc file to use the dev db alias `default`, but it seems like firebase didn't recognize it (the PR workflow failed to deploy). I have changed the rc file to use the actual db name instead of the aliases for now.